### ISSUE TITLE
MessagesSendMultiMedia throws an error if more than 8 files are sent

### ISSUE THREAD

**Issue by itlangzi** [AmarnathCJD/gogram#170]
**Describe the bug**
MessagesSendMultiMedia throws an error if more than 8 files are sent
```
sending MessagesSendMultiMedia: [WORKER_BUSY_TOO_LONG_RETRY] Telegram workers are too busy to respond immediately (code 500)
exit status 1
```

**Comment by AmarnathCJD** [comment]
I'm not sure if this is a telegram's internal issues, or issue with lib implementation

**Comment by itlangzi** [comment]
@AmarnathCJD solved here https://github.com/iyear/tdl/issues/462

**Comment by AmarnathCJD** [comment]
> @AmarnathCJD solved here [iyear/tdl#462](https://github.com/iyear/tdl/issues/462)

where's the commit that solved it .

**Comment by itlangzi** [comment]
> > @AmarnathCJD solved here [iyear/tdl#462](https://github.com/iyear/tdl/issues/462)
> 
> where's the commit that solved it . they just added a retry mechanismm..

yes, this error is telegram internal , and tdl added a retry if this error thrown

**Comment by itlangzi** [comment]
> > @AmarnathCJD solved here [iyear/tdl#462](https://github.com/iyear/tdl/issues/462)
> 
> where's the commit that solved it . they just added a retry mechanismm..

There is another problem. If the photo is larger than 10M, the error PHOTO_SAVE_FILE_INVALID will be reported when uploading

**Comment by AnjanaMadu** [comment]
@AmarnathCJD take https://visibleearth.nasa.gov/images/73751/july-blue-marble-next-generation-w-topography-and-bathymetry

**Comment by AmarnathCJD** [comment]
> > > @AmarnathCJD solved here [iyear/tdl#462](https://github.com/iyear/tdl/issues/462)
> > 
> > 
> > where's the commit that solved it . they just added a retry mechanismm..
> 
> There is another problem. If the photo is larger than 10M, the error PHOTO_SAVE_FILE_INVALID will be reported when uploading

max supported for telegram processed photo size is 10mb, greater than that u have to send it as doc ig

client.SendMedia(
		"roseloverx", "world.topo.bathy.200407.3x5400x2700.png", &tg.MediaOptions{ForceDocument: true},
)
